bedwars-plugin/src/main/java/com/andrei1058/bedwars/BedWars.java

```diff
@@ -302,10 +302,10 @@ public void onEnable() {
                 //registerEvents(new ArenaListeners());
                 ArenaSocket.lobbies.addAll(config.getList(ConfigPath.GENERAL_CONFIGURATION_BUNGEE_OPTION_LOBBY_SERVERS));
                 new SendTask();
-                registerEvents(new AutoscaleListener(), new PrePartyListener(), new JoinListenerBungee());
+                registerEvents(new AutoscaleListener(), new JoinListenerBungee());
                 Bukkit.getScheduler().runTaskTimerAsynchronously(this, new LoadedUsersCleaner(), 60L, 60L);
             } else {
-                registerEvents(new ServerPingListener(), new JoinListenerBungeeLegacy());
+                registerEvents(new ServerPingListener(), new PrePartyListener(), new JoinListenerBungeeLegacy());
```

Comment:
you registered back pre party in a wrong position. move it up in the if autoscale